x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/types.ts

```diff
@@ -72,6 +72,13 @@ export const action = t.exact(
   })
 );
 
+export const responseAction = t.exact(
```

Comment:
Can we share this with the `responseAction` schema in `common`?

Reply:
yes 👍